run_tests

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-PYBOSSA_SETTINGS="./pybossa/settings_test.py" PYBOSSA_REDIS_CACHE_DISABLED=1 nosetests ${@:-test/}
+PYBOSSA_SETTINGS="./pybossa/settings_test.py" PYBOSSA_REDIS_CACHE_DISABLED=1 nosetests --cover-erase ${@:-test/}
```

Comment:
Do we want to change the arguments to nosetests?

Reply:
resolved via 56138ec